### DISCUSSION TITLE
HTML compiler

### DISCUSSION THREAD

**Discussion by jaeming** [ujjwalguptaofficial/mahal#4] [Q&A]
I've been looking through the repo source code and was trying to figure out how things work. I saw reference to mahal-html-compiler but I noticed the repo for that was not public (or maybe it's a typo?). Is that intentionally private? Might make it hard for others to contribute if so.

**Comment by ujjwalguptaofficial** [comment]
Hey, sorry that was by mistake. I was working on all these repos as private and forgot to make this one public. Thanks for reminding :).

**Reply by ujjwalguptaofficial** [reply]
You can view the repo at - https://github.com/ujjwalguptaofficial/mahal-html-compiler
